### ISSUE TITLE
Test benchmark

### ISSUE THREAD

**Issue by voldemortX** [once-3dlanes/once_3dlanes_benchmark#2]
I see the test labels are held-out? Will there be a public benchmark for submitting test set results or it is already there but I did not find? @nick112358

**Comment by once-3dlanes** [comment]
Thanks for your question.
We provide the annotations of the training and validation set of ONCE-3DLanes for the purpose of training.
The annotations of the test set are held-out. In the coming future, we will provide a submission platform for your evaluation results on the test set.

**Comment by voldemortX** [comment]
Thanks. A held-out test set is great. Looking forward to the submission platform.
